src/test/java/org/broadinstitute/hellbender/utils/nio/XorWrapper.java

```diff
@@ -0,0 +1,80 @@
+package org.broadinstitute.hellbender.utils.nio;
+
+import com.google.common.annotations.VisibleForTesting;
+import java.io.IOException;
+import java.nio.ByteBuffer;
+import java.nio.channels.NonWritableChannelException;
+import java.nio.channels.SeekableByteChannel;
+import java.util.function.Function;
+
+/**
+ * A SeekableByteChannel wrapper for testing.
+ */
+public class XorWrapper implements SeekableByteChannel {
```

Comment:
I moved it, but as a result I fear it'll be included in the jar and loaded into every worker every time we run a job.